bin/prpl-server

```diff
@@ -1,3 +1,26 @@
 #!/usr/bin/env node
+
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
 'use strict';
-require('../lib/cli.js')
+
+const cli = require('../lib/cli.js');
+
+try {
+  cli.run(process.argv);
+} catch (e) {
+  console.error(e);
+  process.exit(64);
```

Comment:
I actually chose it because of this exact document! :)

"The author of this document proposes restricting user-defined exit codes to the range 64 - 113"